packages/x-date-pickers-pro/src/SingleInputDateRangeField/tests/editing.SingleInputDateRangeField.test.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import { SingleInputDateRangeField } from '@mui/x-date-pickers-pro/SingleInputDateRangeField';
+import { screen, userEvent, fireEvent } from '@mui/monorepo/test/utils';
+import { expectInputValue } from 'test/utils/pickers-utils';
+import { describeAdapters } from '@mui/x-date-pickers/tests/describeAdapters';
+
+describe('<SingleInputDateRangeField /> - Editing', () => {
+  describeAdapters(
+    'key: Backspace',
+    SingleInputDateRangeField,
+    ({ render, adapter, clickOnInput, selectSection }) => {
+      it('should clear all the sections when all sections are selected and all sections are completed', () => {
+        render(
+          <SingleInputDateRangeField
+            format={adapter.formats.monthAndYear}
+            defaultValue={[adapter.date(), adapter.addYears(adapter.date(), 1)]}
+          />,
+        );
+        const input = screen.getByRole('textbox');
+        clickOnInput(input, 1);
+
+        // Select all sections
+        userEvent.keyPress(input, { key: 'a', ctrlKey: true });
+
+        userEvent.keyPress(input, { key: 'Backspace' });
```

Comment:
Nitpick, what do you think about also testing that `Delete` key works the same? 🤔 
Maybe a bit waste of time and resources to repeat it for all tests, but we could have at least one test in each suite duplicated to ensure this consistency. 🤔

Reply:
I ran all the tests with both keys.
Those tests are super light compared to the DataGrid ones, I think it's worth adding :+1: